### PR TITLE
[WIP] Fixing absence of 9.0 branch.

### LINKED ISSUE #135: Failing tests with Odoo 9.0
I have a repo in git.vauxoo.com and it uses odoo-shippable image to execute the tests it was green until the image was updated, the checking the logs I realized that for some reason it was using odoo 8.0 instead of 9.0 (the few lasts lines):
error

And in the previous one (the is from yesterday):
green

We didn't change the tests nor the code because we're testing the CI system.

Also we notice this:
clone90

Compared with the one that was successfully tested:
good

Regards

### PR DESCRIPTION
**Pull request by LuisAlejandro**
This PR closes #135

### CODE REVIEW

File: odoo-shippable/scripts/library.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ git_clone_copy(){
     WHAT="${3}"
     WHERE="${4}"
     TEMPDIR="$( mktemp -d )"
-    OPTIONS="--depth 30 -b ${BRANCH} -q --single-branch --recursive"
+    OPTIONS="--depth 10 -b ${BRANCH} -q --single-branch --recursive"
```
Comment (nhomar) [inline odoo-shippable/scripts/library.sh]: I do not remember why, but the 30 had a reason --> @moylop260 ?

**Comment by LuisAlejandro** [comment]
This is ready for review @moylop260 @ruiztulio.

**Comment by moylop260** [comment]
Could you review the following script
https://github.com/Vauxoo/maintainer-quality-tools/blob/master/travis/travis_install_nightly#L43?

The big image should fetch the branches to download a cache.
The MQT shoud fetch the branches with or without cache.

Here you are fixing the first option but we will have the same problem using other no-cache repo/branch.

**Comment by ruiztulio** [comment]
@LuisAlejandro I'm still getting some error when it tries to :
error

The env vars in case I'm missing something:
vars

And it is still using 8.0:
version

**Comment by LuisAlejandro** [comment]
😥

**Comment by moylop260** [comment]
@LuisAlejandro 
Please, call me...

**Comment by ruiztulio** [comment]
@moylop260 Just for the record, as I need to continue with my work I did a nasty workaround meanwhile:
nasty

**Comment by moylop260** [comment]
@ruiztulio 
I got it
Thank you

**Comment by moylop260** [comment]
Superseded by https://github.com/Vauxoo/docker-odoo-image/pull/148

**Pull request closed without merging**